index.html

```diff
@@ -4,10 +4,132 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="styles/styles.css">
     <title>Leaders' Week - 2023 - Home</title>
 </head>
 <body>
-    
+    <!-- MENU -->
+    <img src="images/menu.png" alt="menu icon" id="menu-icon">
+    <nav class="menu-box" id="menu-box">
+        <a href="index.html"><img src="images/logo-gc-1.png" alt="Leaders Week Summit 2023 Logo" class="hidden-item logo"></a>
+        <ul>
+            <li><a href="about.html" class="menu-item active">About</a></li>
+            <li><a href="index.html#Program" class="menu-item">Program</a></li>
+            <li><a href="#" class="menu-item">Join</a></li>
+            <li><a href="#" class="menu-item">Sponsor</a></li>
+            <li><a href="#" class="menu-item">News</a></li>
+            <li><a href="index.html" class="menu-item hidden-item red-bordered">LW Company</a></li>
+        </ul>
+    </nav>
```

Comment:
- [ ] It seems like you forgot to add the top (dark) section of the navigation as shown in the original design. so please make sure to add them to make your design look more like the original.
- Original design:
  ![image](https://github.com/lily4178993/leaders-week/assets/104968236/27602dd4-5b0f-44ff-a6eb-c969cb2d1100)
- Your design:
  ![image](https://github.com/lily4178993/leaders-week/assets/104968236/70030f03-6448-43cd-b66c-ae1cbf7f95e3)
- [ ] This applies to both the "Home" & "about" page. ✔